plugins/woocommerce/changelog/51564-fix-various-wc-download-handler-issues

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+WC_Download_Handler buffer handling WC_Download_Handler error default status set to 403 to prevent files from not being served if 404 are cached by a page cache and to ensure the PHP status header matches the downstream reverse-proxy status
```

Comment:
It's a little hard to read, possibly because we're covering different fixes in one single message. May be worth breaking up into multiple messages, or else simplifying a little.